doc/rtd/explanation/format.rst

```diff
@@ -78,6 +76,80 @@ Content found to be gzip compressed will be uncompressed.
 The uncompressed data will then be used as if it were not compressed.
 This is typically useful because user data is limited to ~16384 [#]_ bytes.
 
+``include`` file
+================
+
+This content is an :file:`include` file.
+
+The file contains a list of URLs, one per line. Each of the URLs will be read
+and their content will be passed through this same set of rules, i.e., the
+content read from the URL can be gzipped, MIME multi-part, or plain text. If
+an error occurs reading a file the remaining files will not be read.
```

Comment:
what is meant by "set of rules". and why is it the same for all urls? This might need re-written for dummies like me :P